spark/src/main/scala/com/alibaba/graphar/example/Neo4j2GraphAr.scala

```diff
@@ -0,0 +1,99 @@
+/** Copyright 2022 Alibaba Group Holding Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.graphar.example
+
+import com.alibaba.graphar.datasources._
+import com.alibaba.graphar.graph.GraphWriter
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+object Neo4j2GraphAr {
+
+  def main(args: Array[String]): Unit = {
+    // connect to the Neo4j instance
+    val spark = SparkSession.builder()
+      .appName("Neo4j to GraphAr for Movie Graph")
+      .config("neo4j.url", "bolt://localhost:7687")
+      .config("neo4j.authentication.type", "basic")
+      .config("neo4j.authentication.basic.username", sys.env.get("NEO4J_USR").get)
+      .config("neo4j.authentication.basic.password", sys.env.get("NEO4J_PWD").get)
+      .config("spark.master", "local")
+      .getOrCreate()
+
+    // initialize a graph writer
+    val writer: GraphWriter = new GraphWriter()
+
+    // put movie graph data into writer
+    readAndPutDataIntoWriter(writer, spark)
+
+    // write in graphar format
+    val outputPath: String = args(0)
+    val vertexChunkSize: Long = args(1).toLong
+    val edgeChunkSize: Long = args(2).toLong
+    val fileType: String = args(3)
+
+    writer.write(outputPath, spark, "MovieGraph", vertexChunkSize, edgeChunkSize, fileType)
```

Comment:
Can you help to update the test data?

Reply:
The CI and case is no need to read from gar-test repo anymore.

Reply:
> The CI and case is no need to read from gar-test repo anymore.

We do not need read from gar-test previously, too. The test data in gar-test is used to give a showcase for the data generated by you example.